src/svg/SkSVGDevice.cpp

```diff
@@ -960,6 +1009,61 @@ void SkSVGDevice::drawPath(const SkPath& path, const SkPaint& paint, bool pathIs
     if (pathPtr->getFillType() == SkPathFillType::kEvenOdd) {
         elem.addAttribute("fill-rule", "evenodd");
     }
+
+    if (std::optional<SkBlendMode> mode = paint.asBlendMode()) {
+        std::string blendMode = "";
+        switch(*mode) {
+            case SkBlendMode::kMultiply:
+            blendMode = "multiply";
+            break;
+            case SkBlendMode::kScreen:
+            blendMode = "screen";
+            break;
+            case SkBlendMode::kOverlay:
+            blendMode = "overlay";
+            break;
+            case SkBlendMode::kDarken:
+            blendMode = "darken";
+            break;
+            case SkBlendMode::kLighten:
+            blendMode = "lighten";
+            break;
+            case SkBlendMode::kColorDodge:
+            blendMode = "color-dodge";
+            break;
+            case SkBlendMode::kColorBurn:
+            blendMode = "color-burn";
+            break;
+            case SkBlendMode::kHardLight:
+            blendMode = "hard-light";
+            break;
+            case SkBlendMode::kSoftLight:
+            blendMode = "soft-light";
+            break;
+            case SkBlendMode::kDifference:
+            blendMode = "difference";
+            break;
+            case SkBlendMode::kExclusion:
+            blendMode = "exclusion";
+            break;
+            case SkBlendMode::kHue:
+            blendMode = "hue";
+            break;
+            case SkBlendMode::kSaturation:
+            blendMode = "saturation";
+            break;
+            case SkBlendMode::kColor:
+            blendMode = "color";
+            break;
+            case SkBlendMode::kLuminosity:
+            blendMode = "luminosity";
+            break;
+        }
+        if(!blendMode.empty()) {
+            std::string styleValue = "mix-blend-mode: " + blendMode;
+            elem.addAttribute("style", styleValue.c_str());
```

Comment:
SVGs don't support blend modes as [presentation attributes](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/Presentation), they only work as a style properties.
Figma exports svg blend modes this way, but illustrator exports them as presentation attributes, which fails (even when reimporting them in illustrator itself!).
So I'm using the "style" approach, which works when opening them in the browser and importing them in figma, but it gets lost when importing them in illustrator.

Reply:
What is `mix-blend-mode`? Is this an SVG thing or did we define it somewhere?

Reply:
it's a css/svg thing. That's how blend modes are defined in HTML and SVG
https://developer.mozilla.org/en-US/docs/Web/CSS/mix-blend-mode